apps/piping/app.config.js

```diff
@@ -9,6 +9,8 @@ export default () => ({
   },
   environment: {
     uri: 'https://backend-fusion-data-gateway-test.radix.equinor.com',
+    electricalUri:
+      'https://electrical-consumers-fusion-data-gateway-test.radix.equinor.com',
```

Comment:
These must be updated in the fusion app config before merging. Remind me

Reply:
Added general comment in PR